### DISCUSSION TITLE
[14.0->15.0]  Need step by step migration help

### DISCUSSION THREAD

**Discussion by wangweitung** [OCA/OpenUpgrade#3614] [Q&A]
I am using odoo 14.0 in docker on ubuntu 20.04。
Now I want to migrate to odoo 15.0 in docker。
I searched the issue and google ,but fond no step by step documentation。
I also see https://oca.github.io/OpenUpgrade/ but it's hard for me to understand how to do it .
so can anyone help? very much thanks~

**Comment by pedrobaeza** [comment]
Please ask specific questions. The steps are there: https://oca.github.io/OpenUpgrade/migration_details.html

**Reply by wangweitung** [reply]
> Please ask specific questions. The steps are there: https://oca.github.io/OpenUpgrade/migration_details.html

thanks for your answer。
but I just don't know how to migrate，maybe I should just give up。

**Reply by kevinberger-ch** [reply]
In my opinion the best "step by step" guide: https://www.cybrosys.com/blog/how-to-upgrade-the-odoo-15-database-using-openupgrade
